packages/frontend-2/components/project/page/automations/EmptyState.vue

```diff
@@ -1,48 +1,130 @@
 <template>
-  <div class="flex flex-col gap-y-6 md:gap-y-8">
-    <div class="p-4 flex flex-col gap-y-4 rounded-lg max-w-2xl mx-auto items-center">
-      <div class="gap-y-4 flex flex-col items-center">
-        <div class="text-heading-lg text-foreground">Scale your digital impact</div>
-        <div class="text-body-xs text-foreground-2">
-          Speckle Automate empowers you to continuously monitor your published models,
-          automatically ensuring project data standards, identifying potential design
-          faults, and effortlessly creating delivery artifacts.
-        </div>
-        <div class="flex gap-x-4">
-          <FormButton
-            :disabled="!!creationDisabledMessage"
-            @click="$emit('new-automation')"
-          >
-            New automation
-          </FormButton>
-          <FormButton color="outline" :to="functionsGalleryRoute">
-            View functions
-          </FormButton>
-        </div>
-      </div>
+  <section class="flex flex-col items-center justify-center py-8 md:py-16">
+    <h3 class="text-heading-lg text-foreground">
+      Scale your digital impact with Automate. Let's get you started...
+    </h3>
+    <div class="grid grid-cols-1 lg:grid-cols-3 gap-3 pt-5 mt-4 max-w-5xl">
+      <CommonCard
+        v-for="emptyStateItem in emptyStateItems"
+        :key="emptyStateItem.title"
+        :title="emptyStateItem.title"
+        :description="emptyStateItem.description"
+        :buttons="emptyStateItem.buttons"
+      />
     </div>
-  </div>
+  </section>
 </template>
 <script setup lang="ts">
 import {
   automationFunctionsRoute,
   workspaceFunctionsRoute
 } from '~/lib/common/helpers/route'
 import type { CreateAutomationSelectableFunction } from '~/lib/automate/helpers/automations'
+import type { LayoutDialogButton } from '@speckle/ui-components'
 
-defineEmits<{
+const emit = defineEmits<{
   'new-automation': [fn?: CreateAutomationSelectableFunction]
+  'new-function': []
 }>()
 
-const props = defineProps<{
-  workspaceSlug?: string
-  isAutomateEnabled: boolean
-  creationDisabledMessage?: string
-}>()
+export type AutomateOnboardingAction =
+  | 'create-function'
+  | 'view-functions'
+  | 'create-automation'
+
+const props = withDefaults(
+  defineProps<{
+    workspaceSlug?: string
+    hiddenActions?: AutomateOnboardingAction[]
+    disabledActions?: {
+      action: AutomateOnboardingAction
+      reason: string
+    }[]
+  }>(),
+  {
+    hiddenActions: () => [],
+    disabledActions: () => []
+  }
+)
+
+const router = useRouter()
 
 const functionsGalleryRoute = computed(() =>
```

Comment:
This is showing as unused

Reply:
It is (should be) used on line 93 for `router.push(functionsGalleryRoute.value)`. Maybe a better way to do conditional routes like this?

Reply:
No, it was just flagged as unused in my IDE. But you're right, it is used. Oddly the warning has gone now too. Feel free to ignore, not sure what happened